immersion/kubeflow_pipelines/pipelines/solutions/lab-02_vertex.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<p>you need to give your default service account storage access as below, otherwise the pipeline run fails in a fresh environment. This needs to be run in the Cloud Shell, won't work when run in a notebook as compute service account!</p><pre class="ql-syntax" spellcheck="false">PROJECT_ID=$(gcloud config get-value project)
PROJECT_NUMBER=gcloud projects list --filter="name=$PROJECT_ID" --format="value(PROJECT_NUMBER)" 
gcloud projects add-iam-policy-binding $PROJECT_ID \
    --member="serviceAccount:$PROJECT_NUMBER-compute@developer.gserviceaccount.com" \
    --role="roles/storage.objectAdmin"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Added that as a note that this may need to be run.</p>

Reply:
Here above you do need to include the `PROJECT_ID=...` line too, because else the Cloud Shell does not have that environment variable.

Reply:
Done!